nav-kibana-dev.docnav.json

```diff
@@ -165,6 +165,48 @@
         { "id": "kibVisTypeTimeseriesPluginApi" },
         { "id": "kibVisualizationsPluginApi" }
       ]
+    },
+    {
+      "label": "Operations",
```

Comment:
@spalger is it our intention to merge this initial spike with the operations team menu placed in its current place? Or should we find our final intended location beforehand ?

Reply:
I was think we would merge what we have, we can always update it later